config/Dockerfiles/tests.d/openstack/01_os-server-new

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -xe
+
+# Verify os-server provisioning
+# distros.exclude: none
+# providers.include: openstack
+# providers.exclude: none
+
+DISTRO=${1}
+PROVIDER=${2}
+
+PINFILE="PinFile.${PROVIDER}.yml"
+TARGET="os-server-new"
+
+function clean_up {
+    set +e
+    linchpin -w . -p "${PINFILE}" -v destroy "${TARGET}"
+}
+trap clean_up EXIT SIGHUP SIGINT SIGTERM
+
+pushd docs/source/examples/workspace
+linchpin -w . -p "${PINFILE}" -v up "${TARGET}"
```

Comment:
isn't this covered by general/01_general?

Reply:
That it is, I'll remove this one.